### ISSUE TITLE
I have some issues,may be help me please.

### ISSUE THREAD

**Issue by shify** [qixin1106/iOS8-PersonalVPN-Demo#2]
When I fill in the data, carries on the real machine simulation debugging, did not produce description file and fill in the password and share key

**Comment by qixin1106** [comment]
#define kVPNName @"你的VPN用户名"
#define kServerAddress @"你的VPN地址"

Did you set up "kVPNName" and "kServerAddress"?

**Comment by shify** [comment]
YES,I have set,When I'm really running on NEVPNStatusDisconnected XCode will tell me

**Comment by shify** [comment]
i am waiting for  you when you have free time .
